jwst/extract_1d/ifu.py

```diff
@@ -663,6 +673,9 @@ def image_extract_ifu(input_model, source_type, extract_params):
     n_bkg : ndarray, 1-D, float64
         For each slice, this is the number of pixels that were added
         together to get background.
+
+    x_center, y_center : float
```

Comment:
These have been added to the "Returns" list in the doc string here, but they aren't actually returned by the function.

Reply:
hah that was an oversight. That function is used in the pipeline but I guess it is in the tests.